### DISCUSSION TITLE
Add a posibility to change the color for specific ticks (like gauge)

### DISCUSSION THREAD

**Discussion by ykrapiva** [patrykandpatrick/vico#820] [Ideas]
I'm currently trying to draw ping response time graph.
I would like to have a possibility to further control the color of ticks, for example, if the y-value > 100 then the color should blend with yellow, if > 200 then with red. 
Something like this, i think the actual blen formula cna be left to the implementor.

**Comment by Gowsky** [comment]
Hello @ykrapiva. You can do this by creating a subclass of `VerticalAxis`. We don’t currently plan on adding a dedicated API for this because the use case is rather specific, and the benefit over subclassing would be minimal and not worth the increased complexity.
